python/lsst/verify/bin/jobReporter.py

```diff
@@ -106,14 +105,17 @@ def run(self):
             datasetRefs = list(self.registry.queryDatasets(dataset,
                                collections=self.collection))
             for ref in datasetRefs:
+                names = list(ref.dataId.names)
+                names.append('run')  # "run" must be in the template
+                key_tmpl = '_'.join(['{'+el+'}' for el in names])
```

Comment:
I'm surprised this passes a `flake8` check. Shouldn't it be:
```suggestion
                key_tmpl = '_'.join(['{' + el + '}' for el in names])
```

Reply:
Hmm.  Not sure why flake8 missed it, but I agree your version is more correct